src/index.ts

```diff
@@ -15,7 +15,12 @@ export type KeyDerivationOptions = {
 
 export type EncryptionKey = {
   key: CryptoKey;
-  derivationFunction: KeyDerivationOptions;
+  derivationFunction?: KeyDerivationOptions;
+};
+
+export type ExportedEncryptionKey = {
+  key: JsonWebKey;
+  derivationFunction?: KeyDerivationOptions;
 };
```

Comment:
If we don't include the derivation function metadata in the exported key we will lose important information, that will not allow us to know whether it is a key derived from old or new methods (and we will not know the params used).

This impacts both `exportKey` and `importKey`, as these are the two methods that interacts with this struct